meetings/agenda.md

```diff
@@ -7,3 +7,20 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to discuss
 
+- PHP 8.0 introduction in packages
+
+  As we are about to support PHP 8.0, what is our strategy to do so?
+  - Do we want to support PHP `8.0` or `8.*`? (`php: ^8.0` *vs* `php: ~8.0.0`)
```

Comment:
probably mention keep current minimal version?

```json
php: ^7.1 || ^8.0 
```

or 

```json
php: ^5.6 || ^7.0 || ^8.0 
```

Reply:
I think if we add support for 8, this will be in a minor version, at which point we can drop support for 5.6.

Reply:
I'd say not only PHP 5.6 but everything what is < 7.3 (PHP 7.2 is "alive" for 3 more months only)

Reply:
tbh. even if 7.2 is alive, people can stick to current versions and we probably will backport security fixes if needed. 
I would also propose supporting only actively developed php versions, which will be, at the end of the year, 7.4 and 8.0
If people can stick with legacy php versions, they dont need new features either.